### ISSUE TITLE
Remove quickstart keys from tutorial?

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#976]
The tutorial covers basic keys twice.  It's redundant.
Besides, these keys are one click too far for the impatient user who wouldn't care about the tutorial.

That said, now we have an interactive status bar, beginners don't really need keys anymore.  So I suggest we remove the quickstart keys and add a line in the `*help*` buffer explaining that the buttons in the status bar are clickable (it might not be obvious since they look unfamiliar).

**Comment by aadcg** [comment]
I'd perhaps suggest getting rid of the tutorial altogether, since right now it's basically a subset of the manual. I agree with the rest.

**Comment by Ambrevar** [comment]
The idea behind the separation between the tutorial and the manual is to
provide a quick introduction for newcomers without scaring them away
with too much documentation.

**Comment by aadcg** [comment]
I agree with that. It's just that right now there's little difference.

Then, I don't understand what you want to remove. I guess I didn't get it when you mentioned that the tutorial covers basic keys twice.

**Comment by jmercouris** [comment]
I think duplication of information is OK. If the quickstart keys are shorter than the other descriptions in the tutorial, then we can remove the other descriptions in the tutorial.

Actually the tutorial can probably be even shorter than it is currently (in my opinion).

What do you think?

**Comment by Ambrevar** [comment]
André Alexandre Gomes <notifications@github.com> writes:

> I agree with that. It's just that right now there's little difference.

Between the tutorial and the manual?  The manual _embeds_ the tutorial,
so the first part of the manual is exactly the same as the tutorial,
it's on purpose.

> Then, I don't understand what you want to remove. I guess I didn't get it when you mentioned that the tutorial covers basic keys twice.

The tutorial has both a "Basic controls" and a "Quickstart keys"
section.  Basic controls repeats all the bindings in Quickstart keys,
with more details.

**Comment by aadcg** [comment]
I think the tutorial should be shorter and it should be written in a different style. Right now, it feel like reading documentation. Think about how  the Emacs tutorial is written - it slowly introduces the user to the concepts and bindings in a "friendly" (less opaque) language.

**Comment by Ambrevar** [comment]
John Mercouris <notifications@github.com> writes:

> Actually the tutorial can probably be even shorter than it is currently (in my opinion).

Do you have something specific in mind?
Maybe send a pull request so we can discuss it.

**Comment by aadcg** [comment]
Pierre Neidhardt <notifications@github.com> writes:

> Between the tutorial and the manual? The manual _embeds_ the tutorial,
> so the first part of the manual is exactly the same as the tutorial,
> it's on purpose.

Right and I understand the motivation behind it.  I suggest another
approach above - drawing a parallel with Emacs.

> The tutorial has both a "Basic controls" and a "Quickstart keys"
> section. Basic controls repeats all the bindings in Quickstart keys,
> with more details.

Oh, I see!  Indeed the Quickstart keys feels extra.

**Comment by Ambrevar** [comment]
The problem with a more "friendly" style is that it will almost
certainly make the tutorial longer.

I'm happy to be proven wrong, pull requests welcome! :)

**Comment by aadcg** [comment]
Pierre Neidhardt <notifications@github.com> writes:

> The problem with a more "friendly" style is that it will almost
> certainly make the tutorial longer.

A valid point too.  I think most Nyxt users are powerusers so I don't
think they're the kind of people looking for a "tl;dr" kind of thing.
Anyway, the progress made on the being able to do customizations in the
GUI made it very friendly to everyone.

> I'm happy to be proven wrong, pull requests welcome! :)

For the time being I don't see the point in investing time in re-writing
a new tutorial.  For people who come from Emacs, it's enough to say "a
browser with an Emacs-like interface" and the rest they can figure out
by themselves with muscle-memory and so on.  Other users are also taken
into account due to what I noted above regarding customization.

I can try to help in the future with the tutorial of any documentation,
if you see the need for it :)

**Comment by Ambrevar** [comment]
Thanks for the offer! :)

**Comment by aartaka** [comment]
Is that the time and place to ask for interactive tutorial (:star_struck:) that'll walk the user through all the important features of Nyxt and make them actually use these features? `lisp-url`, the main prerequisite for interactivity, is already there, so we're able to implement it, I believe!

**Comment by Ambrevar** [comment]
You are right, this would be a great idea!

Then we can properly separate our tutorial from the manual.

**Comment by aadcg** [comment]
Artyom Bologov <notifications@github.com> writes:

> Is that the time and place to ask for interactive tutorial (🤩) that'll
> walk the user through all the important features of Nyxt and make them
> actually use these features? lisp-url, the main prerequisite for
> interactivity, is already there, so we're able to implement it, I
> believe!

Could you expand a bit on the interactive tutorial and `lisp-url`?

**Comment by aartaka** [comment]
> Could you expand a bit on the interactive tutorial and `lisp-url`?

Sure. What I meant was to have a series of web-pages created in the same way as `manual` and `tutorial` and leading the user through such Nyxt concepts as buffers (switching, creating, killing them), tree-like history (moving back and forth), and keyboard-driven link hinting (not Nyxt-specific, but quite rare) with the help of these pages. Referencing these pages (well, functions that create these pages, actually) with `lisp-url` is a simplest way to connect them with each other.

Emacs tutorial is plaintext file that user can edit to understand how Emacs works, and we can have the same interactivity. It's just that our medium is not text but webpages!

**Comment by aadcg** [comment]
This issue is related to #2890.
